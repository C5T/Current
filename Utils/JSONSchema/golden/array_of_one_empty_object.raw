{"Array":{"element":{"Object":{"fields":{},"instances":1,"nulls":0}},"instances":1,"nulls":0}}