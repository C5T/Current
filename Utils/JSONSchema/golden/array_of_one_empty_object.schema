// Autogenerated schema inferred from input JSON data.

CURRENT_STRUCT(Schema_Element_Object) {
};

using Schema = std::vector<Schema_Element_Object>;
