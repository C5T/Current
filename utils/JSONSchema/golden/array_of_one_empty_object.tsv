Field	Type	Set	Unset/Null	Values	Details
Schema	Array	1	0
Schema[]	Object	1	0	empty object	
